include/libdragon.h

```diff
@@ -31,6 +31,8 @@
 #include "console.h"
 #include "debug.h"
 #include "joybus.h"
+#include "joybus_accessory.h"
+#include "joypad.h"
```

Comment:
At the moment, `joypad_accessory.h` is not public.

The following are used internally by the Joypad subsystem and are not intended to be exposed:
* `joypad_accessory_detect_async`
* `joypad_rumble_pak_toggle_async`

The following are intended to be public, but require more consideration about usage:
* `joypad_get_transfer_pak_status`
* `joypad_transfer_pak_enable_async`
* `joypad_transfer_pak_load_async`
* `joypad_transfer_pak_store_async`

I don't want to use callbacks because they will be called-back under interrupt. I'm going to keep these internal for now until we can come up with a better strategy for handling asynchronous operations.